sim_module_test_sketch/sim_module_test_sketch.ino

```diff
@@ -1,38 +1,41 @@
 #include <SoftwareSerial.h>
+SoftwareSerial mySerial(3, 2); //SIM800L Tx & Rx is connected to Arduino D3 & D2
 
-//Create software serial object to communicate with SIM800L
-SoftwareSerial mySerial(3, 2);  //SIM800L Tx & Rx is connected to Arduino D3 & D2
-
-void setup() {
-  //Begin serial communication with Arduino and Arduino IDE (Serial Monitor)
```

Comment:
Please revert unnecessary changes like this

Reply:
Fixed.